infra/hooks/postdeploy.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+echo "Configuring EventGrid subscription for API Center"
+
+# Sleep for 2 minutes to allow the function app to be deployed
+sleep 120
+
+functionID=$(az functionapp function show --name ${AZURE_FUNCTION_NAME} \
```

Comment:
If we want to run these scripts outside of the context of an azd comamnd, then we'll need to load the azd env values.  Let's get all the scripts to do that consistently.